config/application.rb

```diff
@@ -37,5 +37,7 @@ class Application < Rails::Application
 
     config.assets.enabled = true
     config.assets.paths << Rails.root.join("uploads")
+
+    config.active_job.queue_adapter = :delayed_job
```

Comment:
We don't use delayed_job anymore do we? This broke it for me anways...
Remove everything related to DJ?